### PR TITLE
OGC feedback comments

### PR DESCRIPTION
**Pull request by ggetz**
Edits to address feedback from OGC comment period. 

Reordered the tiles section, added more explanation for geometric error and relation to SSE, and clarified tileset vs tile geometric error.

@lilleyse

### CODE REVIEW

File: specification/README.md [hunk 1]
```diff
@@ -184,66 +185,41 @@ Note that glTF defines its own node hierarchy, where each node has a transform.
...
-        0,
-        19.4
-      ]
-    },
-    "uri": "2/0/0.b3dm"
-  },
-  "children": [...]
-}
-```
+A tile's geometric error defines the selection metric for that tile. Its value is a nonnegative number specified, in meters, to be used by client implementation to calculate, along with other screen space metrics including the distance from the tile to the camera, the screen size and resolution, the SSE introduced if this tile is rendered and its children are not. If the introduced SEE exceeds the maximum allowed, the tile is refined and its children are considered for rendering.
```
[the remark this answers is not in the document]
Comment (lilleyse) [reply specification/README.md]: > used by client implementation to calculate, along with other screen space metrics including the distance from the tile to the camera, the screen size and resolution, the SSE introduced if this tile is rendered and its children are not

Grammatically this sentence is a little hard to follow.

File: specification/README.md [hunk 2]
```diff
@@ -511,12 +532,9 @@ The top-level object in the tileset JSON has four properties: `asset`, `properti
...
-
-`root` is an object that defines the root tile using the JSON described in the [above section](#tiles).  `root.geometricError` is not the same as the tileset's top-level `geometricError`. The tileset's `geometricError` is the error when the entire tileset is not rendered; `root.geometricError` is the error when only the root tile is rendered.
-
-`root.children` is an array of objects that define child tiles.  Each child tile's content is fully enclosed by its parent tile's `boundingVolume` and, generally, a `geometricError` less than its parent tile's `geometricError`.  For leaf tiles, the length of this array is zero, and `children` may not be defined.
+`geometricError` is a nonnegative number that defines the error, in meters, that determines if the tileset is rendered.  At runtime, the geometric error is used to compute _Screen-Space Error_ (SSE), the error measured in pixels.  If the SSE does not exceed a required minimum, the tileset should not be rendered, and none of its tile should be considered for rendering, see [Geometric error](#geometric-error).
```
Comment (lilleyse) [inline specification/README.md]: `its tile` -> `its tiles`

**Comment by ggetz** [comment]
Thanks for the feedback @lilleyse, updated!

**Comment by lilleyse** [comment]
Looks great, thanks @ggetz.
